x-pack/plugins/triggers_actions_ui/public/application/sections/rule_form/rule_edit.tsx

```diff
@@ -228,30 +229,48 @@ export const RuleEdit = ({
                   <></>
                 )}
                 <EuiFlexItem grow={false}>
-                  <EuiButton
-                    fill
-                    color="success"
-                    data-test-subj="saveEditedRuleButton"
-                    type="submit"
-                    iconType="check"
-                    isLoading={isSaving}
-                    onClick={async () => {
-                      setIsSaving(true);
-                      const savedRule = await onSaveRule();
-                      setIsSaving(false);
-                      if (savedRule) {
-                        onClose(RuleFlyoutCloseReason.SAVED);
-                        if (onSaveHandler) {
-                          onSaveHandler();
-                        }
-                      }
-                    }}
-                  >
-                    <FormattedMessage
-                      id="xpack.triggersActionsUI.sections.ruleEdit.saveButtonLabel"
-                      defaultMessage="Save"
-                    />
-                  </EuiButton>
+                  <EuiFlexGroup alignItems="center">
+                    <EuiFlexItem grow={false}>
+                      <EuiIconTip
+                        type="alert"
+                        position="top"
+                        data-test-subj="changeInPrivilegesTip"
+                        content={i18n.translate(
+                          'xpack.triggersActionsUI.sections.ruleEdit.changeInPrivilegesLabel',
+                          {
+                            defaultMessage:
+                              'Saving this rule will cause a change in privileges and may make the rule behave differently.',
```

Comment:
Just thinking more on this...

> Saving will change the privileges to the rule's available data. This may cause the rule to perform differently.

Throwing out another option here.  I think I'm also getting stuck on 'differently'.  "...differently than previous executions?" (Are we no longer using the term 'executions'?).

cc/ @gchaps @lcawl --> Do you have any thoughts here?

Reply:
I don't mind the current text, but here are some alternatives:
> Saving this rule will change its privileges and thus might change its behavior.

> If you save this rule, it remembers your security privileges and uses them to runs its queries.

Reply:
I'm +1 to `Saving this rule will change its privileges and thus might change its behavior.`

Reply:
A minor tweak:

> Saving this rule will change its privileges and might change its behavior.

Reply:
Works for me.  Thank you for taking a look!

Reply:
Thanks everyone, I've updated the PR (and description screenshot) with the latest https://github.com/elastic/kibana/pull/131738#issue-1228004420.